bigflow/cli.py

```diff
@@ -249,9 +249,27 @@ def _parse_args(project_name: Optional[str], args) -> Namespace:
     _create_project_version_parser(subparsers)
     _create_release_parser(subparsers)
     _create_start_project_parser(subparsers)
+    _create_logs_parser(subparsers)
 
     return parser.parse_args(args)
 
+def _create_logs_parser(subparsers):
+    parser = subparsers.add_parser('logs', description='Returns link leading to logs in GCP Logging.')
+    parser.add_argument('-dc', '--deployment-config-path',
+                        type=str,
+                        help='Path to the deployment_config.py file. '
+                             'If not set, {current_dir}/deployment_config.py will be used.')
```

Comment:
There are utilities to find 'project root', could try to use them instead of 'cwd'?

Reply:
Removed